lib/kitchen/verifier/inspec.rb

```diff
@@ -214,6 +214,12 @@ def runner_options(transport, state = {}, platform = nil, suite = nil) # rubocop
             # default to false until we default to true in inspec
             runner_options[:backend_cache] = false
           end
+          
+          if config[:backend]
+            runner_options[:backend] = config[:backend]
```

Comment:
Are we sure backend is enough? This would only work if we use the user defaults. Is there a case where users configure different credentials for gcp, azure and aws?

Reply:
Hmm. Maybe? :D

Reply:
Very much so - which would actually be the production use case to have different creds

Reply:
I'm not sure what you want to do here. Do you want to remove it?
If so, why treating `config[:backend]` differently than `config[:backend_config]`? Shouldn't they _both_ go?